test/integration/start_stop_delete_test.go

```diff
@@ -74,98 +70,119 @@ func TestStartStop(t *testing.T) {
 		}
 
 		for _, tc := range tests {
-			tc := tc
```

Comment:
is this shaddow reflected in the refactor ?

Reply:
It is a parameter now so I would think so, but haven't checked